index.js

```diff
@@ -303,32 +303,38 @@ Transformer.prototype.renderAsync = function (str, options, locals, cb) {
   }
 };
 Transformer.prototype.renderFile = function (filename, options, locals) {
-  if (typeof this._tr.renderFile === 'function') {
+  if (!this.can('renderFile')) { // *Async, *Client
+    throw new Error('The Transform "' + this.name + '" does not support rendering synchronously.');
```

Comment:
I'm not sure what you are saying here. `!this.can(arg)` means that `arg` doesn't exist, and no fallbacks for `arg` exist either. If `.render` is implemented, `this.can('renderFile')` must be true.

The comment:

```js
// *Async, *Client
```

list the only cases where `this.can('renderFile')` isn't or shouldn't be true.

See https://github.com/jstransformers/jstransformer/blob/renderfile/index.js#L100-L130.